src/custom/state/orders/utils.ts

```diff
@@ -66,6 +66,10 @@ export function classifyOrder(order: OrderMetaData | null): ApiOrderStatus {
   return 'pending'
 }
 
+export function getLimitPrice(order: Order): any {
```

Comment:
return type `Price` gives me:

`Generic type 'Price<TBase, TQuote>' requires 2 type argument(s).`

Tried setting the type as `Price<Token, Token>` without success. Could use some help here (perhaps @alfetopito as you touched this more recently).

Reply:
in re: we should fix the `any` type here.

Reply:
Will give this a try

Reply:
Done, pushed to this branch.